packages/esm-amd-loader/package.json

```diff
@@ -15,6 +15,7 @@
     "lint": "tslint --project . --format stylish",
     "build": "npm run clean && tsc && npm run minify",
     "minify": "babel-minify lib/esm-amd-loader.js --outFile=lib/esm-amd-loader.min.js",
+    "get-size": "npm run build && cat lib/esm-amd-loader.min.js | gzip -9 > esm-amd-loader.min.js.gz && ls -a -l -h esm-amd-loader.min.js.gz && rm esm-amd-loader.min.js.gz",
```

Comment:
`npm run build && gzip -c -9 lib/esm-amd-loader.min.js | wc -c`

Reply:
Ah ok. Can kill some of the cats and pipes still: `npm run build && gzip -9 esm-amd-loader.min.js && ls -l -h esm-amd-loader.min.js.gz && rm esm-amd-loader.min.js.gz`

Reply:
That deletes esm-amd-loader.min.js which causes tests to fail (which is Good and Correct), but that's why I went with the way more roundabout script

Reply:
Oh, there is the `-k` or `--keep` flag to gzip which does the same thing. Whatever though.